tools/format/swf/instance/DynamicText.hx

```diff
@@ -102,6 +102,12 @@ class DynamicText extends TextField {
 			
 		}
 		
+		if (tag.hasFontClass) {
+			
+			trace("TODO: Implement .hasFontClass for DynamicText");
```

Comment:
@jgranick this will probably become important for the same reason as above but we haven't run into this case yet so ok to delay